src/components/common/common-input.scss

```diff
@@ -0,0 +1,21 @@
+@import "style/common/constants";
+@import "style/common/typography";
+
+label {
+  display: flex;
+  flex-direction: column;
+  font-family: "DM Sans Regular";
+  font-size: 1rem;
```

Comment:
In general, when you are developing a website you'll have a custom typographies catalog and then use only those typographies in many places. It's a good practice to have a font style file with all of your typographies declared there and then reuse them in other places.

Reply:
But I have "DM Sans Regular" defined in a file, is that what you are saying?

Reply:
No, e.g. Have you seen when you are using google docs there are different kinds of typographies like Title 1, Title 2, Title 3, normal, paragraph, etc? 

Something similar happens when you are developing a website. There are different kinds of fonts with different properties like: Font Family, Font Size, font weight, letter spacing, etc. To avoid repeating these properties many times around your code, we usually define those typographies once and reuse them many times

![Screenshot 2022-07-27 at 17 13 56](https://user-images.githubusercontent.com/11773865/181363722-d6bc4b52-19a5-4047-9e03-e5096594bad2.png)

Reply:
Oh! got it, thanks!